### ISSUE TITLE
Add TOR proxy to the results

### ISSUE THREAD

**Issue by wuniversales** [searx/searx#1808]
It would be nice to add a tor proxy to the results.
Weboproxy for example, gives the option.

TOR proxy:
https://tor.weboproxy.com/index.php?url=https://www.facebook.com
Canada Proxy:
https://ca.weboproxy.com/index.php?url=https://www.facebook.com

regards

**Comment by unixfox** [comment]
I don't really understand how do you want this to be integrated into Searx, do you want a similar link as the "cached" text next to each result?

**Comment by wuniversales** [comment]
Yes, it could be done like this, or better yet, make a drop-down menu in each result, with the option to view in cache, proxify by tor or proxify with a classic proxy.

Menu example:

http://example.com DuckDuckGo 🠻 <- Here the drop-down menu opens when you mouse over or click

When displaying the menu, the 3 options appear.

Cache 🕐 -> https://web.archive.org/web/http://example.com
Proxify with TOR 🧅 -> https://tor.weboproxy.com/index.php?url=http://example.com
Proxify without TOR 🇨🇦 -> https://ca.weboproxy.com/index.php?url=http://example.com

Even if you wish, you could add more options to the results menu, such as ...

Scan with VirusTotal 🦟 -> https://www.virustotal.com/url/submission/?force=1&url=http://example.com
Search logins with Bugmenot 👽 -> http://bugmenot.com/view/example.com

This option only available for youtube and invidio.us
Download from Youtubepp ⬇️ -> https://www.youtubepp.com/watch?v=abWsiwslrrc

More or less that would be the idea, I hope you like it: D

**Comment by return42** [comment]
Does it make sense to make it configurable?

**Comment by wuniversales** [comment]
> Does it make sense to make it configurable?

Question of tastes, if you want to make it more customizable, yes.
If not nothing

**Comment by virtadpt** [comment]
Configurable would be nice.  Making it an optional flag in REST API calls to Searx would be awesome.  Right now I'm proxying some of my search requests through onion.to for this purpose, but it's not a great solution.

**Comment by unixfox** [comment]
I think it should be configurable because not everyone wants to have a menu with huge number of options for each result.
